client/web/src/enterprise/campaigns/list/CampaignListPage.tsx

```diff
@@ -83,6 +84,43 @@ export const CampaignListPage: React.FunctionComponent<CampaignListPageProps> =
             <p className="text-muted">
                 Run custom code over hundreds of repositories and manage the resulting changesets
             </p>
+            <div className="row">
+                <div className="col-12 col-md-6 mb-2">
+                    <div className="campaign-list-page__intro-card card p-2 h-100">
+                        <div className="card-body d-flex align-items-start">
+                            <SourcegraphIcon className="mr-3 col-2 mt-2" />
+                            <div>
+                                <h4>Campaigns trial</h4>
+                                <p className="text-muted mb-0">
+                                    Campaigns will be a paid feature in a future release. In the meantime, we invite you
+                                    to trial the ability to make large scale changes across many repositories and code
+                                    hosts. If you’d like to discuss use cases and features,{' '}
+                                    <a href="https://about.sourcegraph.com/contact/sales/">please get in touch</a>!
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div className="col-12 col-md-6 mb-2">
+                    <div className="campaign-list-page__intro-card card h-100 p-2">
+                        <div className="card-body">
+                            <h4>New campaigns features in version 3.22</h4>
+                            <ul className="text-muted mb-0 pl-3">
+                                <li>All users can now create campaigns</li>
```

Comment:
As a security person, I would raise my brow here, I guess. Maybe we should reword this a little? (I know it's resolved in the next bullet point, but maybe I would stop reading here and reach out directly)

```suggestion
                                <li>Regular users can publish campaigns using their identity on the code host</li>
```

Reply:
(I don't 100% like my suggestion)

Reply:
@rrhyne

Reply:
I changed the wording a bit so it's clear that "all users can create campaigns" is one thing and that this requires credentials. Happy to accept further input from @rrhyne, but we can always do that in a follow-up.